lib/dynamic_screening_solutions/api.py

```diff
@@ -458,6 +458,9 @@ def get_webhook_topics(self, company_id):
         except Exception:
             webhook_topics = []
 
+        if 'error' in webhook_topics and webhook_topics['error'] is True:
+            webhook_topics = []
```

Comment:
```suggestion
            if webhook_topics.get('error'):
                webhook_topics = []
        except Exception:
            webhook_topics = []

```

We can move it into the `try` block and simply the expression to just do a `.get`

Reply:
Done